packages/devtools_app/lib/src/screens/memory/panes/chart/chart_control_pane.dart

```diff
@@ -83,22 +83,51 @@ class _ChartControlPaneState extends State<ChartControlPane>
     controller.resumeLiveFeed();
   }
 
+  void _clearTimeline() {
+    ga.select(analytics_constants.memory, analytics_constants.clear);
+
+    controller.memoryTimeline.reset();
+
+    // Clear all analysis and snapshots collected too.
+    controller.clearAllSnapshots();
+    controller.classRoot = null;
+    controller.topNode = null;
+    controller.selectedSnapshotTimestamp = null;
+    controller.selectedLeaf = null;
```

Comment:
do we still need all these? It looks like we are now having each memory tab below handle its own clearing?

Reply:
I will clean it up together with analysis tab.